.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,25 +7,25 @@ assignees: ''
 
 ---
 
-**Expected behavior**
+## Expected behavior
 A clear and concise description of what you expected to happen.
 
-**Describe the bug**
+## Describe the bug
 A clear and concise description of what the bug is.
 
-**To Reproduce**
+## To Reproduce
 Steps to reproduce the behavior:
 1. 
 2. 
 3. 
 4. 
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+## Screenshots/output
+If applicable, add screenshots or program output to help explain your problem.
 
-**System Specifications:**
- - Version:
- - Platform:
+## System Specifications:
+ - Software version:
```

Comment:
Is this just relevant to saltproc version, or are there required dependencies that we usually have issues with?

Reply:
e.g. should we ask for serpent versions? openMC versions? numpy?

Reply:
Yeah we should probably specify.
```suggestion
 <!-- At a minimum, include the SaltProc version and versions of software directly related to the bug-->
 - Software version:
```